src/workerd/api/http.h

```diff
@@ -471,9 +471,15 @@ class Fetcher: public jsg::Object {
   jsg::Promise<QueueResponse> queue(
       jsg::Lock& js, kj::String queueName, kj::Array<ServiceBindingQueueMessage> messages);
 
+  jsg::Promise<kj::Maybe<jsg::V8Ref<v8::Value>>> getRpcAndCallRemote(
+      jsg::Lock& js, kj::String methodName, jsg::Optional<kj::Array<jsg::Value>> args=nullptr);
+  // Get an rpc service and call the remote method.
+  // This will evolve once we have a proxy type and can intercept method calls.
+
   JSG_RESOURCE_TYPE(Fetcher, CompatibilityFlags::Reader flags) {
     JSG_METHOD(fetch);
     JSG_METHOD(connect);
+    JSG_METHOD(getRpcAndCallRemote);
```

Comment:
Non-blocking: generally not happy with the idea of adding more and more on `Fetcher` like this, especially since we have uses of `Fetcher` that this will never apply to.

Reply:
Originally planned on putting in `DurableObject`, but tests use `ColoLocalActor` whose `get()` returns `Fetcher`. Added a new type cc @bcaimano since you brought this up too.

https://github.com/cloudflare/workerd/pull/1028/commits/a163e099ba3d2cc6060a1ce5346908c5ab282eda